00_Intro/00_Intro.md

```diff
@@ -51,25 +57,32 @@ You can also access subMIT via JupyterHub, which provides a web-based interface
 
 Python is a high-level programming language that is widely used in scientific computing. It is known for its readability and ease of use. We will use (mostly) Python to write our code.
 
-Two important things:
-1. Various versions of Python exist. A basic one is already installed on the subMIT cluster, which we will use for now. Later, we will use the one determined by the FCC-ee software. Different versions can be installed, for example, with`conda`, a package manager for Python.
-2. Python works with packages, which are downloadable libraries that extend the functionality of Python. These can be installed with `pip`, the Python package installer, or with `conda`, a package manager for Python.
+Some important notes:
+1. The Python programming language is constantly  developped resulting in different versions. The subMIT cluster comes with a pre-installed version of python. This may not be the case on other computing systems.
+2. There are many computing tasks which are useful in many different contexts. For example reading input, matrix multiplications, or plotting. We don't need to implement them all from scratch but can use packages (which are typically very efficient and have some built-in sanity checks).
+3. The two most commonly used tools to install packages are:
+   * [Conda](https://anaconda.org/anaconda/conda): This is a system package manager. It can be used to install python itself (as well as other tools, apps, or programming languages) and python packages.
+   * [Pip](https://pypi.org/project/pip/): This is a python-specific package manager.
+   Both of them are pre-installed on the subMIT cluster.
+4. Not all packages are compatible with all python versions. In order to avoid conflicts when you need different packages for different projects for example, it is **very strongly** adviced to **ALWAYS** install packages within a virtual environment. Then you can easily turn different versions and combinations of packages on or off depending on your needs.
+
+Creating environments and installing packages is beyond the scope of this tutorial. In order to use the FCC-ee software, we use a pre-compiled environment that includes all package you will need for this tutorial.
 
-> *Exercise*: A very simple tutorial for Python can be found in the [subMIT User's Guide](https://submit.mit.edu/submit-users-guide/tutorials/tutorial_1.html). A more in-depth tutorial for python for HEP is provided by the LHCb collaboration [here](https://hsf-training.github.io/analysis-essentials/python/README.html). If you've never worked with Python, go through these tutorials.
+> *Exercise*: A very simple tutorial for Python can be found in the [subMIT User's Guide](https://submit.mit.edu/submit-users-guide/tutorials/tutorial_1.html). A more in-depth tutorial for python for HEP is provided by the HSF collaboration [here](https://hsf-training.github.io/analysis-essentials/python/README.html). If you've never worked with Python, go through these tutorials.
```

Comment:
I think, we may want to be a bit more specific here. This is a very much to ask over the holidays.